[2021-05-23 04:39:35,254] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: marketvol.connect 2021-05-23T04:39:19.265757+00:00 [queued]>
[2021-05-23 04:39:35,279] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: marketvol.connect 2021-05-23T04:39:19.265757+00:00 [queued]>
[2021-05-23 04:39:35,281] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-05-23 04:39:35,282] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2021-05-23 04:39:35,282] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-05-23 04:39:35,301] {taskinstance.py:887} INFO - Executing <Task(DummyOperator): connect> on 2021-05-23T04:39:19.265757+00:00
[2021-05-23 04:39:35,308] {standard_task_runner.py:53} INFO - Started process 3252 to run task
[2021-05-23 04:39:35,382] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: marketvol.connect 2021-05-23T04:39:19.265757+00:00 [running]> 208e7b2250b3
[2021-05-23 04:39:35,415] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=marketvol, task_id=connect, execution_date=20210523T043919, start_date=20210523T043935, end_date=20210523T043935
[2021-05-23 04:39:45,189] {logging_mixin.py:112} INFO - [2021-05-23 04:39:45,187] {local_task_job.py:103} INFO - Task exited with return code 0
